tests/osaka/eip7825_transaction_gas_limit_cap/test_tx_gas_limit_transition_fork.py

```diff
@@ -35,12 +40,11 @@ def test_transaction_gas_limit_cap_at_transition(
     Before timestamp 15000: No gas limit cap (transactions with gas > 2^24 are valid)
     At/after timestamp 15000: Gas limit cap of 2^24 is enforced
     """
-    sender = pre.fund_eoa()
     contract_address = pre.deploy_contract(
-        code=Op.SSTORE(0, Op.ADD(Op.SLOAD(0), 1)) + Op.STOP,
+        code=Op.SSTORE(Op.TIMESTAMP, Op.ADD(Op.SLOAD(0), 1)) + Op.STOP,
```

Comment:
> Since there is no transaction gas limit cap before Osaka is activated, so two successful transaction should lead to 2 increments in the storage. However, the final result only increase the value by one. I am still exploring the possible issue here.

It looks like you are loading the value from slot `0` which is never updated. I'm just starting to look here but it looks like we need the `SLOAD` to load from `Op.TIMESTAMP` slot. If you change this you do get a final value of `2` in the `post` for the contract storage at slot `14_999`.

```suggestion
        code=Op.SSTORE(Op.TIMESTAMP, Op.ADD(Op.SLOAD(Op.TIMESTAMP), 1)) + Op.STOP,
```